base/constants.go

```diff
@@ -89,6 +89,13 @@ const (
 	StacktraceOnAPIErrors = false
 )
 
+const (
+	SyncFnErrorMissingRole          = "missing role"
+	SyncFnErrorAdminRequired        = "admin required"
+	SyncFnErrorWrongUser            = "wrong user"
+	SyncFnErrorMissingChannelAccess = "missing channel access"
```

Comment:
Should we prefix these with something like 'sg' to make it less likely that users include a custom throw that matches this term?